CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ Possible log types:
 
 ### Unreleased
 
+- [changed] Update redis to 0.15 and r2d2\_redis to 0.13 (#105)
```

Comment:
Do you want to mention the MSRV change as well?

Reply:
Well we don't have a MSRV according to our README, that's why I didn't mention it.

Reply:
Ah, true.